app/views/events/_applicants_overview.html.erb

```diff
@@ -41,18 +40,54 @@
   <div class="container-fluid no-padding">
     <div class="btn-group" role="group">
       <%= link_to "Alle annehmen",
-                  events_path, :class => 'btn btn-default'%>
+          events_path, :class => 'btn btn-default'%>
 
       <%= link_to "Alle drucken",
-                  events_path, :class => 'btn btn-default'%>
+          events_path, :class => 'btn btn-default'%>
     </div>
     <div class="btn-group pull-right" role="group">
-      <%= link_to "Zusagen verschicken",
-                  events_path, :class => 'btn btn-default'%>
-
-      <%= link_to "Absagen verschicken",
-                  events_path, :class => 'btn btn-default'%>
+      <% if (!@event.applications_classified?) %>
+          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.unclassified_applications_left' %>">
+            <%= button_tag t('.sending_acceptances'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
+          </div>
+          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.unclassified_applications_left' %>">
+            <%= button_tag t('.sending_rejections'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
+          </div>
+      <% elsif (@event.compute_free_places < 0)%>
+          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.maximum_number_of_participants_exeeded' %>">
+            <%= button_tag t('.sending_acceptances'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
+          </div>
+          <div class="tooltip-wrapper disabled has-tooltip" data-toggle="tooltip" title="<%= t '.maximum_number_of_participants_exeeded' %>">
+            <%= button_tag t('.sending_rejections'), type: 'button', class: 'send-emails-button btn btn-default', disabled: true %>
+          </div>
+      <% else %>
+          <%= button_tag t('.sending_acceptances'), type: 'button', class: 'btn btn-default', data: {toggle: 'modal', target: '#send-emails-modal', title: t('.sending_acceptances'), list: @event.accepted_applications_emails} %>
+          <%= button_tag t('.sending_rejections'), type: 'button', class: 'btn btn-default', data: {toggle: 'modal', target: '#send-emails-modal', title: t('.sending_rejections'), list: @event.rejected_applications_emails} %>
+      <% end %>
     </div>
   </div>
 
 </div>
+
+<!-- Bootstrap Modal for sending emails -->
+<div id="send-emails-modal" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+
+    <!-- Modal content-->
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+        <h4 class="modal-title">Emails verschicken</h4>
```

Comment:
Can the Tag be empty or filled in the English Version? I think the Code should be consitently in English.